apps/prairielearn/src/tests/instructorAssessmentGroups.test.ts

```diff
@@ -114,6 +114,26 @@ describe('Instructor group controls', () => {
     group2RowId = groupRow.attr('data-test-group-id');
   });
 
+  step('can create a group with an instructor', async () => {
+    const getResponse = await fetchCheerio(instructorAssessmentGroupsUrl, {});
+    const csrfToken = extractAndSaveCSRFToken({}, getResponse.$, '[name="add-group-form"]');
+
+    const response = await fetchCheerio(instructorAssessmentGroupsUrl, {
+      method: 'POST',
+      body: new URLSearchParams({
+        __csrf_token: csrfToken,
+        __action: 'add_group',
+        group_name: 'TestGroupWithInstructor',
+        // Add instructor to the group
+        uids: 'dev@illinois.edu',
```

Comment:
@nwalters512 is the hardcoded UID here ok or should I use the config parameter (or some other means) to get the instructor UID?

Reply:
Hardcoding is fine, I think.